makeDist

```diff
@@ -330,81 +333,81 @@ unlink("pod2htmi.tmp");
+            s/__VERSION__/$Version/g;
+            s/__RELEASEDATE__/$ReleaseDate/g;
+            s/^([#*\s]+)Copyright\s+\(C\)\s+(\d{4})-\d{4}\s+Craig\s+Barratt/$1Copyright (C) $2-$year  Craig Barratt/i;
+            if ( $file =~ /BackupPC\.html$/ ) {
+                #
+                # fixup for conf links
+                #
+                if ( /<dt id="Conf-/ ) {
+                    s/<dt id=("Conf-.*)>\$Conf\{([^}]*)}/
 			defined($ConfName{$2})
 			    ? "\L<dt id=\"$ConfName{$2}\">\E\$Conf{$2}<\/a>"
 			    : "<dt id=$1>\$Conf{$2}"/eg;
                 } else {
-		    s/\$Conf\{([^}]*)}/
+                    s/\$Conf\{([^}]*)}/
 			defined($ConfName{$1})
 			    ? "\L<a href=\"#$ConfName{$1}\">\E\$Conf{$1}<\/a>"
 			    : "\$Conf{$1}"/eg;
-		}
+                }
                 s{^(<dt id="_conf.*</dt>)}{<strong>$1</strong>};
-		s/^<DD>/<DD><P>/;
-		s/^<li><\/li>/<li>/;
+                s/^<DD>/<DD><P>/;
+                s/^<li><\/li>/<li>/;
                 #
                 # add diveder lines before <h1>
                 #
                 s{^<h1}{<hr />\n<h1}g;
 
-	    }
+            }
             if ( $lineCnt == 0 && m{^#!/usr/local/bin/perl} ) {
                 #
                 # configure.pl overwrites the perl path on install, but do this to keep the
                 # distribution path backward compatible.
                 #
                 s{/usr/local/bin/perl}{/usr/bin/perl};
             }
-	    if ( /__CONFIGPOD__/ ) {
-		print OUT $ConfPod;
-	    } elsif ( /^use lib ".*BackupPC\/lib";/
-		    || /^use lib "\/home\/pcbackup\/install\/lib";/ ) {
-		print OUT "use lib \"__INSTALLDIR__/lib\";\n";
-	    } elsif ( $file =~ /Lib.pm/ && /^(\s*\$topDir\s*=\s*)'.*'(\s*if\s.*)/ ) {
-		print OUT "$1'__TOPDIR__'$2\n";
-	    } elsif ( $file =~ /Lib.pm/ && /^(\s*\$installDir\s*=\s*)'.*'(\s*if\s.*)/ ) {
-		print OUT "$1'__INSTALLDIR__'$2\n";
-	    } elsif ( $file =~ /Lib.pm/ && /^(\s*ConfDir\s*=\>\s*\$confDir eq.*)'.*'(.*)/ ) {
-		print OUT "$1'__CONFDIR__'$2\n";
+            if ( /__CONFIGPOD__/ ) {
+                print OUT $ConfPod;
+            } elsif ( /^use lib ".*BackupPC\/lib";/
+                || /^use lib "\/home\/pcbackup\/install\/lib";/ ) {
+                print OUT "use lib \"__INSTALLDIR__/lib\";\n";
+            } elsif ( $file =~ /Lib.pm/ && /^(\s*\$topDir\s*=\s*)'.*'(\s*if\s.*)/ ) {
+                print OUT "$1'__TOPDIR__'$2\n";
+            } elsif ( $file =~ /Lib.pm/ && /^(\s*\$installDir\s*=\s*)'.*'(\s*if\s.*)/ ) {
+                print OUT "$1'__INSTALLDIR__'$2\n";
+            } elsif ( $file =~ /Lib.pm/ && /^(\s*ConfDir\s*=\>\s*\$confDir eq.*)'.*'(.*)/ ) {
+                print OUT "$1'__CONFDIR__'$2\n";
             } elsif ( $file =~ /Lib.pm/ && /^(\s*my \$useFHS\s*=\s*)\d;/ ) {
                 print OUT "${1}0;\n";
             } elsif ( $file =~ /Lib.pm/ && /(.*Version *=> .*)'[\w\d\.]+',/ ) {
```

Comment:
BTW, After reformatting this substitution doesn't match that`bless` function in lib/BackupPC/Lib.pm
I am afraid we should thoroughly check all substitutions in makeDist :(

Reply:
Same as above, fixed with f3ac5a25a9d68fe467c6531a9931bb3cfc8b511a